src/utils/daos_hdlr.h

```diff
@@ -111,6 +135,11 @@ struct cmd_args_s {
 	daos_obj_id_t		oid;
 	daos_prop_t		*props;		/* --properties cont create */
 
+	/* Container datamover related */
+	struct dm_args		*dm_args;	/* datamover arguments */
+	struct fs_copy_stats	*fs_copy_stats;	/* fs copy stats */
+	bool 		  	 fs_copy_posix; /* fs copy to POSIX */
```

Comment:
(style)  please, no space before tabs

Reply:
I dunno what this is on about. `cat -vet utils/daos_hdlr.h` shows me this:
```
^Ibool^I^I^I fs_copy_posix; /* fs copy to POSIX */$
$
```